share/eds/testing.go

```diff
@@ -465,7 +465,14 @@ func BenchGetSampleFromAccessor(
 
 type quadrantIdx int
 
-var quadrants = []quadrantIdx{1, 2, 3, 4}
+const (
+	q1 quadrantIdx = iota + 1 // top-left: original data quadrant
```

Comment:
Do we use these const somewhere? Looks like a completely unrelated change.

Reply:
Yep, should be removed as in https://github.com/celestiaorg/celestia-node/pull/4091#discussion_r2025102367